.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,40 @@
+{
+  "configurations": [
+    {
+      "name": "Linux",
+      "includePath": [
+        "${workspaceFolder}/code/include/**"
+      ],
+      "defines": [],
+      "compilerPath": "gcc",
+      "cStandard": "c17",
+      "cppStandard": "c++20",
+      "intelliSenseMode": "linux-gcc-x64"
+    },
+    {
+      "name": "Mac",
+      "includePath": [
+        "${workspaceFolder}/code/include/**"
+      ],
+      "defines": [],
+      "macFrameworkPath": [],
+      "compilerPath": "/usr/bin/clang++",

```

Comment:
Similar to the other platform, do we need the absolute path here?

Reply:
Ah! Apologies - I was doing a search for `/usr/bin/g` and missed this one 🤦 Prefix removed and tested from https://github.com/orx/orx/pull/104/commits/02566c48b78253d358034ff25e2628f96d596890